### PR TITLE
Create aaa.svelte

### PR DESCRIPTION
**Pull request by liimee**

### CODE REVIEW

File: aaa.svelte [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<script>
+  var name = 'aaaaaaaaaaaaaaaaa';
```
Comment (Benricheson101) [inline aaa.svelte]: Could you make this add only one `a` at a time, so it's consistent with the rest of the files? See [aaa.html](https://github.com/Benricheson101/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/blob/aaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaa.html)

Comment (liimee) [reply aaa.svelte]: oh oops yeah

File: aaa.svelte [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <script>
-  var name = 'aaaaaaaaaaaaaaaaa';
+  var name = 'a';
   setInterval(()=>name+=name, 1);
```
Comment (Benricheson101) [inline aaa.svelte]: `name += 'a'` would work better.

Comment (liimee) [reply aaa.svelte]: done

**Comment by Benricheson101** [comment]
Perfect, thanks :D

**Pull request merged**